java-frontend/src/main/java/org/sonar/java/ast/JavaAstScanner.java

```diff
@@ -54,22 +51,22 @@ public JavaAstScanner(ActionParser<Tree> parser, @Nullable SonarComponents sonar
   }
 
   public void scan(Iterable<File> files) {
-    ProgressReport progressReport = new ProgressReport("Report about progress of Java AST analyzer", TimeUnit.SECONDS.toMillis(10));
-    progressReport.start(Lists.newArrayList(files));
+//    ProgressReport progressReport = new ProgressReport("Report about progress of Java AST analyzer", TimeUnit.SECONDS.toMillis(10));
+//    progressReport.start(Lists.newArrayList(files));
 
-    boolean successfullyCompleted = false;
+//    boolean successfullyCompleted = false;
     try {
       for (File file : files) {
         simpleScan(file);
-        progressReport.nextFile();
+//        progressReport.nextFile();
       }
-      successfullyCompleted = true;
+//      successfullyCompleted = true;
     } finally {
-      if (successfullyCompleted) {
-        progressReport.stop();
-      } else {
-        progressReport.cancel();
-      }
+//      if (successfullyCompleted) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)